model/clusters_mgmt/v1/cluster_resource.model

```diff
@@ -75,6 +75,17 @@ resource Cluster {
 		target MachinePools
 	}
 
+	// Initiates cluster hibernation; While hibernating a cluster will not consume any cloud provider infrastructure
+	// but will be counted for quota.
+    method Hibernate {
```

Comment:
Make sure to use tabs instead of spaces for indentation.

Reply:
Done. thanks